vagrant/config/versions.yaml

```diff
@@ -99,3 +99,4 @@ installers:
       - 'centos9-stream'
       - 'debian11'
       - 'ubuntu2004'
+      - 'ubuntu2204'
```

Comment:
In some days we will need ubuntu2404 :-)

Reply:
https://github.com/orgs/theforeman/projects/13/views/1 is what had to be done for 22.04, feel free to start the work :)

Reply:
Thanks, I will write a note internally